api_v3/lib/types/KalturaExportToCsvOptions.php

```diff
@@ -11,10 +11,15 @@ class KalturaExportToCsvOptions extends KalturaObject
 	 * @var string
 	 */
 	public $format;
+	/**
+	 * @var KalturaEntryType
+	 */
+	public $typeEqual;
```

Comment:
@TalDubovKaltura  I may be nitpicking here, but if I understand the feature correctly, the field name typeEqual seems like a misnomer, or at least misleading. 
Generally, you want to indicate to the export engine that you would like it to tack additional columns onto the report in specific cases. It's important to note that this option will NOT force the engine to actually only return media entries. Please add a doc comment to this property here && on the API object explaining its purpose.

Reply:
@hilak Done, doc been added.
When I added it, I was aware of this issue and actually debated what to do, but a doc should be good enough.
and one more thing, "the API object explaining its purpose" is not equal to "doc comment to this property"?

Reply:
@TalDubovKaltura  sorry, I thought I'd left the comment on the core object. I edited my comment.